### ISSUE TITLE
No Data Found

### ISSUE THREAD

**Issue by drakblack** [Kyusung4698/PoE-Overlay#509]
# 🐞 bug report

**Describe the bug**
Price Evaluation is not working , try with alot of different item same.
i try aswell to reinstall poe overlay, still same.

![image](https://user-images.githubusercontent.com/8472334/77847283-5873dd00-71bc-11ea-9220-d7eb8d16d82c.png)

## 🌍  Your Environment

- OS:  e.g. Win7 x64
- Version:  e.g. 0.6.9 portable
- PoE:  e.g. Steam 3.10.0/e English

**Comment by Odin099** [comment]
Do you have in the settings enabled the Item Type?
Evaluate > Default Search Settings > Item Type
if yes turn it off and try again

**Comment by drakblack** [comment]
Hi, thanks for the fast answer, item type was already disable. try to enable it and disable it.
but it still dont work

**Comment by Odin099** [comment]
atm the trade site is pretty much down mb thats why

**Comment by ericsium** [comment]
I don't think this is due to the trade site being down.  I just downloaded this and tried ctrl-D on 'quill rain short bow'.  For some reason it initiated the search with item level and *all mods* selected.  I unselected everything but item level, still 'no results'.  I then clicked on 'open in browser' and saw plenty of results.  Maybe something changed with the latest update that might have affected how you're making the queries?
